lib/vscode/src/vs/server/node/channel.ts

```diff
@@ -397,7 +397,8 @@ export class TerminalProviderChannel implements IServerChannel<RemoteAgentConnec
 			case '$onPtyHostStartEvent': return this.ptyService.onPtyHostStart || Event.None;
 			case '$onPtyHostUnresponsiveEvent': return this.ptyService.onPtyHostUnresponsive || Event.None;
 			case '$onPtyHostResponsiveEvent': return this.ptyService.onPtyHostResponsive || Event.None;
-
+			case '$onPtyHostRequestResolveVariablesEvent': return this.ptyService.onPtyHostRequestResolveVariables || Event.None; 
+			
```

Comment:
Nit: extra blank line and a bit of trailing whitespace. :smile:

Reply:
Should be fixed now 🤞